addons/test_mail/tests/test_mail_gateway.py

```diff
@@ -1535,3 +1537,49 @@ def test_message_cc_update_no_old(self):
                                 cc='cc2 <cc2@example.com>, cc3@example.com', target_model='mail.test.cc')
         cc = email_split_and_format(record.email_cc)
         self.assertEqual(sorted(cc), ['"cc2" <cc2@example.com>', 'cc3@example.com'], 'new cc should have been added on record (unique)')
+
+
+@tagged('mail_gateway')
+class TestMailThread(MailCommon):
+    def setUp(self):
+        super(TestMailThread, self).setUp()
+        self.mail_thread = self.env['mail.thread']
+        self.mail_alias = self.env['mail.alias']
+        self.mail_message = self.env['mail.message']
+        self.ir_model = self.env['ir.model']
+
+    def test_message_route_reply_model_none(self):
+        email_from = '"test" <test@gmail.com>'
+        email_to  = email_from
+        message_id = f'<test{time.time()}-openerp@{socket.gethostname()}>'
+        to_create = {
+            'email_from': email_from,
+            'body': '<p>test</p>',
+            'message_id': message_id,
+            'message_type': 'email',
+            'model': None,
+            'res_id': None,
+        }
+        self.mail_message.create(to_create)
+        self.mail_alias.create({'alias_name': 'test', 'alias_model_id': self.ir_model._get('res.partner').id})
```

Comment:
Fyi

I just remembered the other way of doing this, which I mentionned a while back. You can do `self.env.ref('base.model_res_partner')`. But I think querying ir.model directly is better, personally.